app/views/layouts/_environmental_warning.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if Rails.env.review_app? %>
```

Comment:
I think this has to be `ENV["REVIEW_APP"].present?` unfortunately. The `Rails.env.review_app?` will only work if I create an env for that and set the `RAILS_ENV` to `review_app`, i believe.